telemetry/aiwrapper_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Microsoft. All rights reserved.
+package telemetry
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-container-networking/aitelemetry"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateAITelemetryHandle(t *testing.T) {
+	tests := []struct {
+		name          string
+		aiConfig      aitelemetry.AIConfig
+		disableAll    bool
+		disableMetric bool
+		disableTrace  bool
+		wantErr       bool
+	}{
+		{
+			name:          "disable telemetry",
+			aiConfig:      aitelemetry.AIConfig{},
+			disableAll:    false,
+			disableMetric: true,
+			disableTrace:  true,
+			wantErr:       true,
+		},
+		{
+			name:          "empty aiconfig",
+			aiConfig:      aitelemetry.AIConfig{},
+			disableAll:    true,
+			disableMetric: true,
+			disableTrace:  true,
+			wantErr:       true,
+		},
+	}
+
+	for _, test := range tests {
+		tt := test
```

Comment:
for test, just do `tt := tt`

Reply:
@rbtr  Initially I did it, but it looks a little confused. Any critical reasons to stick `tt := tt` ?

```go
for _, tt := range tests {
		tt := tt
```

Reply:
it's just conventional to assign to the same name, since you're not going to use the var in the outer scope
https://github.com/golang/go/wiki/CommonMistakes